samples/star-wars-api/HttpHandlers.fs

```diff
@@ -38,22 +42,28 @@ module HttpHandlers =
     let private graphQL (next : HttpFunc) (ctx : HttpContext) =
         task {
 
+            let logger = ctx.RequestServices.CreateLogger moduleType
+
             // TODO: validate the result
             let toResponse documentId =
```

Comment:
Let's copy my new implementation from Risk.
Try to execute scenarios that are possible (you can try to test them using React samples).